providers/openfeature-provider-flagd/src/openfeature/contrib/provider/flagd/provider.py

```diff
@@ -56,6 +58,8 @@ def __init__(  # noqa: PLR0913
         retry_backoff_max_ms: typing.Optional[int] = None,
         retry_grace_period: typing.Optional[int] = None,
         cert_path: typing.Optional[str] = None,
+        default_authority: typing.Optional[str] = None,
+        grpc_credentials: typing.Optional[grpc.ChannelCredentials] = None,
```

Comment:
any reason why to call them `grpc_credentials` here and not `channel_credentials`?

Reply:
Sorry, an oversight. Thanks for catching!